mindsdb/integrations/handlers/jira_handler/jira_handler.py

```diff
@@ -0,0 +1,237 @@
+from typing import Optional
+from collections import OrderedDict
+
+import pandas as pd
+import duckdb
+import json
+from atlassian import Jira
+
+from mindsdb_sql import parse_sql
+from mindsdb.integrations.libs.base import DatabaseHandler
+from mindsdb.utilities.log import get_log
+from mindsdb_sql.parser.ast.base import ASTNode
+
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+
+log = get_log()
+
+
+class JiraHandler(DatabaseHandler):
+    """
+    This handler handles connection and execution of the Airtable statements.
+    """
+
+    name = 'jira'
+
+    def __init__(self, name: str, connection_data: Optional[dict], **kwargs):
+        """
+        Initialize the handler.
+        Args:
+            name (str): name of particular handler instance
+            connection_data (dict): parameters for connecting to the database
+            **kwargs: arbitrary keyword arguments.
+        """
+        super().__init__(name)
+        self.parser = parse_sql
+        self.dialect = 'jira'
+        self.connection_data = connection_data
+        self.kwargs = kwargs
+
+        self.connection = None
+        self.is_connected = False
+
+    def __del__(self):
+        if self.is_connected is True:
+            self.disconnect()
+
+    def connect(self) -> StatusResponse:
+        """
+        Set up the connection required by the handler.
+        Returns:
+            HandlerStatusResponse
+        """
+
+        if self.is_connected is True:
+            return self.connection
+        
+        jira_instance = Jira(
+            url=self.connection_data['jira_url'],
+            username=self.connection_data['user_id'],
+            password=self.connection_data['api_key'],
+            cloud=True)
+        results = jira_instance.jql(self.connection_data['jira_query'])
+        df = pd.json_normalize(results["issues"])
+        fields_name= ["key", "fields.summary","fields.status.name", "fields.reporter.name","fields.assignee.name","fields.priority.name"]
+        df = df[fields_name]
+        df = df.rename(columns={'key': 'Ticket_No', 'fields.summary': 'Summary','fields.reporter.name':'Reporter','fields.assignee.name':'Assignee','fields.priority.name':'Priority','fields.status.name':'Status'})
+
+
+        globals()[self.connection_data['table_name']] = df
```

Comment:
Do we need globals() here?

Reply:
I am not access this data frame data without globals  in another functions